packages/ai-bot/helpers.ts

```diff
@@ -9,6 +9,7 @@ import type {
 } from 'https://cardstack.com/base/room';
 import { MatrixEvent, type IRoomEvent } from 'matrix-js-sdk';
 
+//TODO: Move patchCard system message out of ai bot
```

Comment:
@IanCal any opinions on where to move this

Reply:
I think this should be passed over in the messages, and sit on the host side. I'm happy this being a single value right now.

When we get to skills, we'll be passing over multiple things (one per prompt) that will get concatenated - so I think we should start by just having it fully host side first.

Then we can put them in their own events and refer to them, so that we're not sending them repeatedly (which will let us turn them on and off easily too).

To make the code not have huge text blocks in I put them in text files and imported them https://github.com/cardstack/boxel/pull/1154/files#diff-a01a1054cd83bd46b0774400aa1946007fd9d0ac66b53ab9bfd8c6e3906bd9bf and https://github.com/cardstack/boxel/pull/1154/files#diff-caa19c46420eab3696e16709e1c81f80a08012a962d9059fa5d265047c3f5ea5R55

Reply:
Maybe not host app but runtime-common helpers. In a file like u suggested? Some ai-specific stuff is already there

Reply:
Hmmm I didn't split it out into txt files. I think over-engineering for now. I moved the system message to runtime-common and matrix-service imports it

Reply:
Cool, they'll end up in base at some point when we have skills.

Reply:
"not in the bot" is the key thing here imo